api/src/main/java/com/velocitypowered/api/command/CommandManager.java

```diff
@@ -34,11 +36,42 @@
   void unregister(String alias);
 
   /**
-   * Attempts to execute a command from the specified {@code cmdLine}.
+   * Calls CommandExecuteEvent and attempts to execute a command from the specified {@code cmdLine}
+   * sync.
    *
    * @param source the command's source
    * @param cmdLine the command to run
    * @return true if the command was found and executed, false if it was not
    */
   boolean execute(CommandSource source, String cmdLine);
+
+  /**
+   * Attempts to execute a command from the specified {@code cmdLine} sync
```

Comment:
Velocity has no concept of `sync`. More accurately this could say `Attempts to execute a command using the specified {@code cmdLine} in a blocking fashion.`